### PR TITLE
Remove toNumber and parseInt from bn tests

### LINKED ISSUE #414: Test Review: unify how we test bn
Unify how we test bn (assert-bignumber vs Number.parseInt() )

### PR DESCRIPTION
**Pull request by leomassazza**
Fix #414

### CODE REVIEW

**Comment by leomassazza** [comment]
Also reordered some requires

File: packages/core-modules/test/contracts/modules/CommsMixin.test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-const assert = require('assert/strict');
+const { ethers } = hre;
+const assert = require('assert');
```
Comment (mjlescano) [inline packages/core-modules/test/contracts/modules/CommsMixin.test.js:2]:
```suggestion
const assert = require('assert/strict');
```

I like to use the `assert/strict` by default, as it uses the Strict comparisons by default and shows better diffs on error: https://nodejs.org/api/assert.html

File: packages/core-modules/test/contracts/modules/RouterCoverage.test.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
+const { ethers } = hre;
 const assert = require('assert');
```
Comment (mjlescano) [inline packages/core-modules/test/contracts/modules/RouterCoverage.test.js:2]:
```suggestion
const assert = require('assert/strict');
```

File: packages/synthetix-main/test/contracts/modules/SNXTokenModule.test.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-const assert = require('assert');
 const { ethers } = hre;
+const assert = require('assert');
```
Comment (mjlescano) [inline packages/synthetix-main/test/contracts/modules/SNXTokenModule.test.js:2]:
```suggestion
const assert = require('assert/strict');
```

File: packages/synthetix-main/test/contracts/modules/SynthsModule.test.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
-const assert = require('assert');
 const { ethers } = hre;
+const assert = require('assert');
```
Comment (mjlescano) [inline packages/synthetix-main/test/contracts/modules/SynthsModule.test.js:2]:
```suggestion
const assert = require('assert/strict');
```

**Review by mjlescano — APPROVED** [review]

File: packages/core-modules/test/contracts/modules/CommsMixin.test.js (hunk 1 above)
Comment (mjlescano) [reply packages/core-modules/test/contracts/modules/CommsMixin.test.js:2]: Moved this to a separate issue: https://github.com/Synthetixio/synthetix-v3/issues/426

File: packages/core-contracts/test/contracts/token/ERC20.test.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 const { ethers } = hre;
 const assert = require('assert');
 const assertRevert = require('@synthetixio/core-js/utils/assert-revert');
-const bn = require('@synthetixio/core-js/utils/assert-bignumber');
+const assertBn = require('@synthetixio/core-js/utils/assert-bignumber');
```
Comment (ajsantander) [inline packages/core-contracts/test/contracts/token/ERC20.test.js:4]: 💯

**Review by ajsantander — APPROVED** [review]
Crá

**Pull request merged**